src/Utils/request/handleResponse.ts

```diff
@@ -15,7 +15,7 @@ export default function handleResponse(
   // 5xx errors
   if (res.status >= 500 && res.status < 600) {
     console.error("Server error", res);
-    notify?.Error({ msg: "Something went wrong...!" });
+    notify?.Error({ msg: error?.detail || "Something went wrong...!" });
     return;
   }
```

Comment:
you could delete this block too.

line 35 will handle this case the same way right?

Reply:
yeah I noticed that but the thing is that in line 17, there is "console.error" statement, which makes this block different from the block in line 29.

Reply:
If you will say to delete the console.error statement, then I will remove this block also.

Reply:
the console error can be dropped. it shouldn't be an issue

Reply:
Ok, sure. I'll make the requested changes.

Reply:
I've made the changes, please review.